inc/saiqueue.h

```diff
@@ -258,6 +280,19 @@ typedef enum _sai_queue_attr_t
      */
     SAI_QUEUE_ATTR_PFC_CONTINUOUS_DEADLOCK_STATE,
 
+    /**
+     * @brief Queue discard action
+     *
+     * Action to be taken upon packet discard due to
+     * buffer profile configuration, WRED configuration
+     * or any other queue attribute.
+     *
+     * @type sai_queue_packet_discard_action_t
+     * @flags CREATE_AND_SET
+     * @default SAI_QUEUE_PACKET_DISCARD_ACTION_NONE
+     */
+    SAI_QUEUE_ATTR_PACKET_DISCARD_ACTION,
```

Comment:
so, again, let's discuss whether this drop action should be a part of buffer profile or queue

Reply:
Put it on a buffer profile and not relevant to WRED